images/kindnetd/Dockerfile

```diff
@@ -27,6 +27,6 @@ COPY . .
 RUN go build -o /go/bin/kindnetd .
 
 # STEP 2: Build small image
-FROM scratch
+FROM gcr.io/google-containers/debian-iptables-amd64:v11.0.2
```

Comment:
this should be `debian-iptables-${GOARCH}:v11.0.2`?

it gives me an error using it

Reply:
We have to pass through GOARCH as an ARG both before and after FROM, otherwise it won't be available (more details in the docker docs for this, on mobile or would link)